src/models/CollectionFlow.ts

```diff
@@ -0,0 +1,12 @@
+import { CollectionItem } from '@/models/CollectionItem'
+
+export type CollectionFlow = CollectionItem<'flow'> & {
+  returns: string,
+  examples: string[],
+  documentationUrl: string,
+  installCommand: string,
+  parameters: unknown,
```

Comment:
should this be of type `SchemaValues` to match how parameters are typed on Deployments?

Reply:
I purposefully left it `unknown` because it's not something I intend on using and I couldn't confirm that it was the same contract as `SchemaValues`.

Reply:
Lots of parameters we use are also Record<string, unknown>

Reply:
it actually might be the same type as `Schema` rather than `SchemaValue`, not sure without looking further into it. I think keeping it loose should be fine, and if I run into any issues when I use it for creating a deployment, we can revisit then.